Makefile

```diff
@@ -327,14 +327,26 @@ build-qe-napi:
 	cargo build --package query-engine-node-api --profile $(PROFILE)
 
 build-qe-wasm:
-ifndef $(NIX)
+ifneq ($(strip $(NIX)),)
 	@echo "Building wasm engine on nix"
 	rm -rf query-engine/query-engine-wasm/pkg
 	nix run .#export-query-engine-wasm query-engine/query-engine-wasm/pkg 0.0.0
 else
 	cd query-engine/query-engine-wasm && ./build.sh
 endif
 
+.PHONY: measure-qe-wasm
+measure-qe-wasm: build-qe-wasm	
+	@cd query-engine/query-engine-wasm/pkg; \
+	gzip -c query_engine_bg.wasm | wc -c | awk '{$$1/=(1024*1024); printf "%.3fMB\n", $$1}' > temp_size.txt; \
+	if [ ! -f size.txt ]; then \
+		echo "0MB" > size.txt; \
+	fi; \
+	echo "Previous size: `cat size.txt`"; \
+	echo "Current size: `cat temp_size.txt`"; \
```

Comment:
Non-blocking nit: can we call this `.size_previous.txt` and `.size_current.txt`?
I feel the current names for these text files could be a source of confusion.

Reply:
Also, let's note that this only measures the gzip'd `wasm` size without considering the necessary JavaScript glue code. On the other hand, `wasm-pack pack` creates a `tar.gz` file that takes the glue code into account as well.

Reply:
How much code does that account for? zipped? single digit KB?